README.md

```diff
@@ -61,7 +61,7 @@ Currenlty only IST/PST/PDT/UTC by default. But user can set major timezones in t
 versions.
 eg.
 ```
-> export MAJOR_TIMEZONES=["UTC", "Asia/Kolkata"]
```

Comment:
I meant 

export MAJOR_TIMEZONES=["UTC","Asia/Kolkata"]
Please check the comment above, we cannot have space in the array.

Reply:
I see, updated with appropriate changes.